config/manager/manager.yaml

```diff
@@ -32,7 +32,7 @@ spec:
             - name: OPERATOR_NAME
               value: "mongodb-kubernetes-operator"
             - name: AGENT_IMAGE # The MongoDB Agent the operator will deploy to manage MongoDB deployments
-              value: quay.io/mongodb/mongodb-agent:10.19.0.6562-1
+              value: quay.io/mongodb/mongodb-agent:10.27.0.6772-1
```

Comment:
this version of the operator allows us to provide the dummy configuration

Reply:
💯